src/main/resources/templates/superAdminDashboard.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+
```

Comment:
Don't we need to add also "xmlns:th="http://www.thymeleaf.org"?

Reply:
I didn't add it, letting our teammate, who would start to work with it, add necessary namespace. I think both will work, but "xmlns:th="http://www.thymeleaf.org/" is better.